official/resnet/resnet_run_loop.py

```diff
@@ -374,6 +379,16 @@ def input_fn_eval():
     eval_results = classifier.evaluate(input_fn=input_fn_eval,
                                        steps=flags.max_train_steps)
     print(eval_results)
+    if benchmark_logger is not None:
+      # The eval_results result contains 3 entries:
```

Comment:
Since the logger is not used before this, may as well instantiate it here under if flags.benchmark_log_dir, and not have two separate conditionals.

Reply:
Done.